url/src/origin.rs

```diff
@@ -93,7 +94,11 @@ impl Origin {
             Origin::Tuple(ref scheme, ref host, port) => {
                 let host = match *host {
                     Host::Domain(ref domain) => {
+                        #[cfg(feature = "idna")]
                         let (domain, _errors) = domain_to_unicode(domain);
```

Comment:
Nit: Do `idna::domain_to_unicode` and remove the `use` above

Reply:
Done